presidio-analyzer/tests/test_ip_recognizer.py

```diff
@@ -21,10 +21,13 @@ def entities():
         # IPv4 tests
         ("microsoft.com 192.168.0.1", 1, ((14, 25),), ((0.6, 0.81),),),
         ("my ip: 192.168.0", 0, (), (),),
-        # IPv6 tests  TODO IPv6 regex needs to be fixed
-        # ("microsoft.com 684D:1111:222:3333:4444:5555:6:77", 1, ((14, 46),), ((0.59, 0.81),),),  # noqa: E501
-        # ("my ip: 684D:1111:222:3333:4444:5555:6:77", 1, ((7, 39),), ((0.79, "max"),),),  # noqa: E501
+        # IPv6 tests
+        ("microsoft.com 684D:1111:222:3333:4444:5555:6:77", 1, ((14, 47),), ((0.6, 0.81),),),  # noqa: E501
+        ("my ip: 684D:1111:222:3333:4444:5555:6:77", 1, ((7, 40),), ((0.6, 0.81),),),  # noqa: E501
         ("684D:1111:222:3333:4444:5555:77", 0, (), (),),
+        ("2345:0425:2CA1:0000:0000:0567:5673:23b5", 1, (), (),),  # noqa: E501
+        ("2345:0425:2CA1::0567:5673:23b5", 1, (), (),),
+        ("::", 0, (), (),),  # Should actually be valid, but probably not at a high score  # noqa: E501
```

Comment:
So would any `::` be detected as ip?

Reply:
Still, it might have lots of false positives. Not sure if we want to treat is as a true positive IP. Can we separate this specific case from all the others and give low score as you suggested?

Reply:
ok